docs/reference/errors-and-warnings/NU5128.md

```diff
@@ -0,0 +1,80 @@
+---
+title: NuGet Warning NU5125
+description: NU5125 Warning code
+author: zivkan
+ms.date: 09/06/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5128
+---
+
+# NuGet Warning NU5128
```

Comment:
we should likely mention that this is being fired a bit too much in 5.3 (package type != dependency and a few other cases) -- and how to workaround.